### PR TITLE
Update readme.md

### PR DESCRIPTION
**Pull request by bkmgit**
Add link to mxnet ruby bindings

I want to contribute to this list and help the maintainers, so:

- [X] I've accepted the terms of the `CC0` License;
- [X] I've added the topic `rubyml` to the contributed repository if appropriate.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -156,6 +156,7 @@ programming languages with appropriate bindings for Ruby.
 - [ruby-dnn](https://github.com/unagiootoro/ruby-dnn) - Simple deep learning for Ruby.
 - [torch-rb](https://github.com/ankane/torch-rb) - Ruby bindings for [LibTorch](https://github.com/pytorch/pytorch)
   using [rice](https://github.com/jasonroelofs/rice).
+- [mxnet](https://mxnet.apache.org/) - Ruby bindings for [mxnet](https://github.com/mrkn/mxnet.rb).
```
Comment (arbox) [inline readme.md]: @bkmgit , could you please exchange the links: Ruby bindings project first, upstream project last.

Comment (bkmgit) [reply readme.md]: @arbox Thanks. Done

File: readme.md [hunk 2]
```diff
@@ -154,8 +154,9 @@ programming languages with appropriate bindings for Ruby.
   Deep learning framework for Ruby.
 - [tensorflow](https://github.com/somaticio/tensorflow.rb) - Ruby bindings for [TensorFlow](https://www.tensorflow.org/).
 - [ruby-dnn](https://github.com/unagiootoro/ruby-dnn) - Simple deep learning for Ruby.
-- [torch-rb](https://github.com/ankane/torch-rb) - Ruby bindings for [LibTorch](https://github.com/pytorch/pytorch)
+- [torch-rb](https://github.com/ankane/torch-rb) - Ruby bindings for [LibTorch](https://github.com/pytorch/pytorch).
```
Comment (arbox) [inline readme.md:157]: Why do we need a dot here in the middle of a sentence?

Comment (bkmgit) [reply readme.md:157]: All other entries seem to have a full stop, eg lines 155, 154 etc. It seemed that line 157 is missing one.

**Comment by arbox** [comment]
@bkmgit ,
thank you for your contribution!

**Pull request merged**